### ISSUE TITLE
[QUESTION] I have a question about making categories.

### ISSUE THREAD

**Issue by ChanhuiSeok** [cotes2020/jekyll-theme-chirpy#21]
Hello. Thank you for providing a nice theme.
Everything works fine except one thing within the theme. One is the issue with categories.

I have read the readme file carefully, and have modified all the urls etc in the config.yml file to suit my github address.
And when I post something new, the category name I created is displayed correctly.
That is, 'https://(*my_url*)/tabs/categories/' works fine.

However, if I click on the generated category title, I get a 404 error page.
In other words, it doesn't connect to 'https://(*my_url*)/categories/(name_of_category)/'.

![capture1](https://user-images.githubusercontent.com/45116772/72216592-1bf1f700-3566-11ea-8f07-e6a07170e907.PNG)
![capture2](https://user-images.githubusercontent.com/45116772/72216593-1c8a8d80-3566-11ea-9d9b-9af5a368366f.PNG)
(There is no access to the 'Studying' category, too.)

I would really appreciate it if you could help me with this.
Thank you very much.
